app/src/components/form/attribute_input.rs

```diff
@@ -56,8 +56,9 @@ pub fn single_attribute_input(props: &SingleAttributeInputProps) -> Html {
     html! {
         <div class="row mb-3">
             <label for={props.name.clone()}
-                class="form-label col-4 col-form-label">
-                {&props.name}{":"}
+                class="form-label col-4 col-form-label"
+                title={props.name.clone()}>
+                {props.name[0..1].to_uppercase() + &props.name[1..].replace("_", " ")}{":"}
```

Comment:
You don't have to do it in this PR, but I'd like to have a little (i) tooltip with the technical name of the field, as well as any LDAP aliases (not currently part of the API)

Reply:
Something like this, already enabled by the title= on the line above? :grin: 
![image](https://github.com/user-attachments/assets/abcafb76-5ac4-4fd3-82e1-9dc4d9036484)

Reply:
That's already great, but I was thinking a more visible "ⓘ", potentially with a dismissable modal? To be mobile-friendly

Reply:
Ooh... I see. That makes the display commit worthy of its very own draft PR :joy: